News Theme Development.md

```diff
@@ -753,144 +527,123 @@ The resizerKey will only accessed via this helper. For security, this helper is
 
 ## Troubleshooting Dependencies
 
-
 ### Finding Your Bearings
 
 #### Check you're in top-level directory
-`pwd`
--> /Users/user/sites/fusion-news-theme-blocks
+
+`pwd` -> /Users/user/sites/fusion-news-theme-blocks
 
 #### Ensure you have latest version
-`git branch`
--> should highlight with * stable
 
-if not,
-`git checkout stable`
+`git branch` -> should highlight with \* stable
+
+if not, `git checkout stable`
 
-fetch latest
-`git fetch`
+fetch latest `git fetch`
 
-pull those, updating your files
-`git pull`
+pull those, updating your files `git pull`
 
-### Clearing House 
+### Clearing House
 
-Ensure all node modules are cleared: 
+Ensure all node modules are cleared:
 
 #### Delete lingering top-level modules
+
 `rm -rf node_modules/`
+
 - should be a bit of a delay if they're there
 - also will autocomplete if they're there and you press tab after `nod`
 
 #### Clear out blocks level deps
 
 `npx lerna clean`
+
 - Deletes blocks' node modules
 - Press y to confirm removing node modules
 
 #### Install top-level
+
 `npm i`
+
 - Installs from top-level `package.json`
 
 #### **DOM-PARSER NOT FOUND**
-- That's something in default output type (so SangHee) that needs to be installed
-`cd blocks/default-output-block`
+
+- That's something in default output type that needs to be installed 
+
+`cd blocks/default-output-block` 
+
 `npm i`
+
 - installs dom parser and others required to run using linking workflow
 
 ### See what's there
 
 #### Check top-level deps
 
 `npm ls react`
+
 - Checks which version of react installed
 - if there's issues with intersections, you'll see it there
-- Search github for the version error, usually common 
+- Search github for the version error, usually common
 
 ### Check published requirements
+
 `npm view @wpmedia/video-player-block`
+
 - Will show address and version
 
-`npm view @wpmedia/video-player-block` 
+`npm view @wpmedia/video-player-block`
+
 - Will show latest version, more concise
 
-### Open Questions
-
-- Why do we need top-level local file dependencies at all
-  - Hoisting up the dependencies is something lerna does, but that yarn workspaces could help
-- What license.md should we have?
-- What about versioning block dependencies? Using `latest` may just be easiest for depending on other blocks
-
-### LocalDev Tips/Playbook
-- If your are running locally with `useLocal` set to true in `blocks.json` and you are noticing that webpack is not picking 
-up your changes in the blocks repo and recompiling, try the following: 
-    1) First ensure you do have `useLocal` set to true.
-    2) Remember in order for Fusion to link to your local working blocks repository, you need to start fusion with 
-        the `--links` or `-l` flag.  Ex: `fusion start theme --links` or `fusion start theme -l`
-    3) Shut down Fusion.
-    4) Run `docker image prune`
-    5) Restart Docker
-    6) Restart Fusion.
-
-- Before publishing, creating a bundle, or right after creating a new branch:
```

Comment:
Would some of these be worth keeping?

Reply:
this is outdated `First ensure you do have `useLocal` set to true.`. Most of it is available in fusion cli docs we link to